crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -31,6 +31,8 @@ def prepare
         service_object = CrowbarService.new(Rails.logger)
 
         service_object.prepare_nodes_for_crowbar_upgrade
+        ret, message = Crowbar::Repository.disable_by(version: 6)
+        raise message unless ret == 200
```

Comment:
I could extend the options hash to fetch service packs (now or later on when we need it)